pl_examples/domain_templates/computer_vision_fine_tuning.py

```diff
@@ -0,0 +1,440 @@
+        return self.__dataloader(train=False)
+
+    @staticmethod
+    def add_model_specific_args(parent_parser):
+        parser = argparse.ArgumentParser(parents=[parent_parser])
+        parser.add_argument('--backbone',
+                            default='resnet50',
+                            type=str,
+                            metavar='BK',
+                            help='Name (as in ``torchvision.models``) of the feature extractor')
+        parser.add_argument('--epochs',
+                            default=15,
+                            type=int,
+                            metavar='N',
+                            help='total number of epochs',
+                            dest='nb_epochs')
+        parser.add_argument('--batch-size',
+                            default=8,
+                            type=int,
+                            metavar='B',
+                            help='batch size',
+                            dest='batch_size')
+        parser.add_argument('--gpus',
+                            type=int,
+                            default=1,
+                            help='number of gpus to use')
+        parser.add_argument('--lr',
+                            '--learning-rate',
+                            default=1e-2,
+                            type=float,
+                            metavar='LR',
+                            help='initial learning rate',
+                            dest='lr')
+        parser.add_argument('--lr-scheduler-gamma',
+                            default=1e-1,
+                            type=float,
+                            metavar='LRG',
+                            help='Factor by which the learning rate is reduced at each milestone',
+                            dest='lr_scheduler_gamma')
+        parser.add_argument('--num-workers',
+                            default=6,
+                            type=int,
+                            metavar='W',
+                            help='number of CPU workers',
+                            dest='num_workers')
+        parser.add_argument('--train-bn',
+                            default=True,
+                            type=bool,
+                            metavar='TB',
+                            help='Whether the BatchNorm layers should be trainable',
+                            dest='train_bn')
+        parser.add_argument('--milestones',
+                            default=[5, 10],
+                            type=list,
+                            metavar='M',
+                            help='List of two epochs milestones')
+        return parser
+
+
+def main(hparams: argparse.Namespace) -> None:
+    """Train the model.
+
+    Args:
+        hparams: Model hyper-parameters
+
+    Note:
+        For the sake of the example, the images dataset will be downloaded
+        to a temporary directory.
+    """
+
+    with TemporaryDirectory(dir=hparams.root_data_path) as tmp_dir:
```

Comment:
I guess we want to keep the output folder

Reply:
@jbschiratti ^^

Reply:
The folder in which the data was downloaded is deleted after the experiment. If you think we should leave the data untouched after the example has run, I can make another PR to fix this  :-)